src/dso_api/dynamic_api/views/wfs.py

```diff
@@ -50,6 +50,15 @@
 )
 
 
+def dataset_has_geometry_fields(dataset) -> bool:
+    for table in dataset.schema["tables"]:
+        for field in table["schema"]["properties"].values():
+            ref_type = field.get("$ref")
+            if ref_type and ref_type.startswith("https://geojson.org/schema/"):
+                return True
+    return False
```

Comment:
```suggestion
    return any(
        field.get("$ref", "").startswith("https://geojson.org/schema/")
        for table in dataset.schema["tables"]
        for field in table["schema"]["properties"].values()
    )
```

Reply:
@lbam deze loopt door alle tabellen en velden, terwijl oude zal bij eerste match stoppen.

Reply:
Nee, any stopt bij de eerste match:

```
>>> def values():
...     while True:
...         yield True
... 
>>> any(values())
True
```

Reply:
Ah, zeker, hier hebben we met generators te makken.

Reply:
Misschien is het een idee om van deze functie op termijn een @cached_property van Dataset te maken?

Reply:
Zat ik ook over na te denken. DatasetSchema is misschien een betere kandidaat. Aan de andere kant is deze expressie nu zo simpel dat het nauwelijks de moeite waard is...